### PR TITLE
DBZ-7305 Update Db2 doc for snapshot modes alignment

### PR DESCRIPTION
**Pull request by mfvitale**
related to: https://github.com/debezium/debezium-connector-db2/pull/142

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/db2.adoc [hunk 1]
```diff
@@ -2623,14 +2679,84 @@ For example, +
...
+ifdef::community[]
+`custom`:: The connector performs a snapshot query according to the implementation specified by the xref:db2-property-snapshot-snapshot-query-mode-custom-name[`snapshot.query.mode.custom.name`] property, which defines a custom implementation of the `io.debezium.spi.snapshot.SnapshotQuery` interface. +
+endif::community[]
+
+This setting enables you to manage snapshot content in a more flexible manner compared to using the xref:db2-property-snapshot-select-statement-overrides[`snapshot.select.statement.overrides`] property.
+
+ifdef::community[]
+|[[db2-property-snapshot-snapshot-query-mode-custom-name]]<<db2-property-snapshot-query-mode-custom-name, `+snapshot.query.mode.custom.name+`>>
+|No default
+| When xref:db2-property-snapshot-query-mode[`snapshot.query.mode`] is set to `custom`, use this setting to specify the name of the custom implementation provided in the `name()` method that is defined by the 'io.debezium.spi.snapshot.SnapshotQuery' interface.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/db2.adoc:2757]:
```suggestion
| When xref:db2-property-snapshot-query-mode[`snapshot.query.mode`] is set to `custom`, use this setting to specify the name of the custom query implementation provided in the `name()` method that is defined by the 'io.debezium.spi.snapshot.SnapshotQuery' interface.
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
@mfvitale A few small suggestions. Otherwise, LGTM. Thanks!

**Comment by mfvitale** [comment]
@roldanbob thanks! Proposed changes addressed.

**Review by roldanbob — COMMENTED** [review]
LGTM! Thanks.

**Pull request merged**